copier/__main__.py

```diff
@@ -0,0 +1,4 @@
+from copier.cli import CopierApp
+
+if __name__ == "__main__":
+    CopierApp.run()
```

Comment:
This seems OK, just please add some test asserting that we can call copier this way.

Reply:
Added a test in a new commit. Feel free to squash instead.